src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -121,6 +121,7 @@ public static function register($mode = 0)
                     $msg = $parsedMsg['deprecation'];
                     $class = $parsedMsg['class'];
                     $method = $parsedMsg['method'];
+                    $isWeak = DeprecationErrorHandler::MODE_WEAK === $mode || (DeprecationErrorHandler::MODE_WEAK_VENDORS === $mode && $isVendor = $inVendors($parsedMsg['triggering_file']));
```

Comment:
shouldn't $isVendor be initialized to false before this line?

Reply:
then L124: `$isWeak = $isWeak || ...`?

Reply:
self answering :)

> shouldn't $isVendor be initialized to false before this line?

not needing as logically this wouldn't make a different, but might still help the reader

> then L124: $isWeak = $isWeak || ...?

really not sure about that (might instead break the fix, you'll tell :) )

Reply:
Yeah that'd break the fix. The point is $isWeak is wrong at this point. Can always add a comment.